fallout3.asl

```diff
@@ -2,13 +2,7 @@ state("Fallout3")
 {
     bool isLoading : 0xC76CE8;
 	bool introDone : 0xC771D0;
-}
-update
-{
-   vars.isLoading = false;
-   if ((current.isLoading) || (!current.introDone)) {
-        vars.isLoading = true;
-    }
+	int vault : 0xC78E28;
```

Comment:
What's this for, is it needed?

Reply:
My bad, I was testing for an autosplitter and forgot to remove it